staging/src/k8s.io/apimachinery/pkg/util/proxy/upgradeaware_test.go

```diff
@@ -162,6 +164,7 @@ func TestServeHTTP(t *testing.T) {
 		notExpectedRespHeader []string
 		upgradeRequired       bool
 		expectError           func(err error) bool
+		overrideRequestHost   bool
```

Comment:
```suggestion
		useLocationHost   bool
```

Reply:
@liggitt I changed it like you suggested